oauthproxy_test.go

```diff
@@ -1748,6 +1748,9 @@ func TestRequestSignature(t *testing.T) {
 
 func TestGetRedirect(t *testing.T) {
 	opts := baseTestOptions()
+	opts.ReverseProxy = true
```

Comment:
I just realized, we need this in the test table and we need some tests with `ReverseProxy` set to `false` resulting in any `X-Forwarded-*` headers being ignored for redirect processing.

Reply:
Hmm, how should this be done, any ideas?

Reply:
I think I got it.